common/src/main/resources/runtime.properties

```diff
@@ -23,8 +23,9 @@
 
 *.falcon.replication.workflow.maxmaps=5
 *.falcon.replication.workflow.mapbandwidth=100
+*.falcon.feed.workflow.mapreduce.map.memory.mb=512
```

Comment:
Please use the same naming convention as the other properties .. falcon.replication.workflow.map.memory.mb.